great_expectations_provider/operators/great_expectations.py

```diff
@@ -112,6 +112,8 @@ class GreatExpectationsOperator(BaseOperator):
     :type return_json_dict: bool
     :param use_open_lineage: If True (default), creates an OpenLineage action if an OpenLineage environment is found
     :type use_open_lineage: bool
+    :param database: If provided, overwrites the default database provided by the connection
```

Comment:
Can we not get these values from the connection extras? 
I feel in general the connection should be self sufficient and exposing these parameters in the operators is separating related things(to the connection) out. Anything that is provided in the extras can override the values for database/schema, etc, no? 

WDYT?

Reply:
From the docs https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable/connections/postgres.html it doesn't look like database is a valid extra.

Reply:
You could get the database from `self.conn.schema`. The Airflow has database name in the web UI, but it's saved as 'schema` column.